shared/chat/conversation/messages/message-popup/exploding/index.tsx

```diff
@@ -164,14 +164,7 @@ class ExplodingPopupHeader extends React.Component<Props, State> {
 }
 
 const ExplodingPopupMenu = (props: Props) => {
-  const header = {
-    style: {
```

Comment:
was this needed?

Reply:
It was ignored by the FloatingMenu, so I hope not :P